web/lib/plugins/seneca-dojo-preloader.js

```diff
@@ -6,21 +6,26 @@
 const translater = require('../fn/i18n-translate');
 const languages = require('country-language');
 
-module.exports = function (request, cb) {
+module.exports = (request, cb) => {
   const preloaded = {};
   const defaultLanguage = 'en_US';
-  request.seneca.act({ role: 'cd-dojos',
+  request.seneca.act({
```

Comment:
act are pattern based : the first 2 define which message, the third is the params.
I wouldn't split the first 2, like, really.